innersource-portal.md

```diff
@@ -0,0 +1,54 @@
+## Title 
+
+* InnerSource Portal
```

Comment:
Very minor: if there is just text in a section (like a one-line title in Title), it's probably better not to put that text as a bullet.